sdk/resourcemanager/compute/armcompute/setup_test.go

```diff
@@ -0,0 +1,137 @@
+package armcompute_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/arm"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/recording"
+	"github.com/stretchr/testify/require"
+	"hash/fnv"
+	"net/http"
+	"os"
+	"testing"
+	"time"
+)
+
+var pathToPackage = "sdk/resourcemanager/compute/armcompute"
```

Comment:
```suggestion
var pathToPackage = "sdk/resourcemanager/compute/armcompute/testdata"
```
Using a `testdata` directory ensures this data isn't added to the pkg.go.dev site and downloaded by customers when they do `go get`